Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -256,12 +258,12 @@ struct RGroupData {
       const {  // compute the canonical smiles for the attachments
     std::string s;
     for (const auto &it : smilesSet) {
-      s += it;
+      s += "."+it;
```

Comment:
I don't think this makes valid smiles.  I think you need

```
{
  if(s.size()) s += ".";
  s += it;
}
```

Reply:
eh, details.
The previous smiles was also wrong. ;-)

Reply:
errr, I mean: good point! I'll fix it!